salt/metalk8s/kubeadm/init/certs/ca.sls

```diff
@@ -0,0 +1,80 @@
+{%- from "metalk8s/map.jinja" import ca with context %}
+
+{%- set ca_server = salt['mine.get']('*', 'kubernetes_ca_server') %}
+
+{#- Check if we have no CA server or only current minion as CA #}
+{%- if not ca_server or ca_server.keys() == [grains['id']] %}
+
+include:
+  - .installed
+
+Create CA private key:
+  x509.private_key_managed:
+    - name: /etc/kubernetes/pki/ca.key
+    - bits: {{ ca.cert.private_key_size }}
+    - user: root
+    - group: root
+    - mode: 600
+    - makedirs: True
+    - dir_mode: 755
+    - require:
+      - pkg: Install m2crypto
+
+Generate CA certificate:
+  x509.certificate_managed:
+    - name: /etc/kubernetes/pki/ca.crt
+    - signing_private_key: /etc/kubernetes/pki/ca.key
+    - CN: kubernetes
+    - keyUsage: "critical digitalSignature, keyEncipherment, keyCertSign"
+    - basicConstraints: "critical CA:true"
+    - days_valid: {{ ca.cert.days_valid }}
+    - user: root
+    - group: root
+    - mode: 644
+    - makedirs: True
+    - dir_mode: 755
+    - require:
+      - x509: Create CA private key
+
+# TODO: Find a better way to advetise CA server
+Advertise CA in the mine:
+  module.wait:
+    - name: mine.send
```

Comment:
According @ramanduh live comment,
```
use_superseded:
  - module.run
``` 
So we need to use the new `module.run` format